contrib/sync_db_pools/codegen/src/lib.rs

```diff
@@ -41,6 +41,11 @@ use proc_macro::TokenStream;
 ///
 ///     Retrieves a connection wrapper from the configured pool. Returns `Some`
 ///     as long as `Self::fairing()` has been attached.
+//!
```

Comment:
We should also change "two" to "three" up above while we're here.

Reply:
Thanks, fixed in 3e0a6df